tests/bgp/bgp_helpers.py

```diff
@@ -118,3 +135,35 @@ def verify_all_routes_announce_to_bgpmon(duthost, ptfhost):
         if route not in bgpmon_routes:
             return False
     return True
+
+def remove_bgp_neighbors(duthost, asic_index):
+    """
+    Remove the bgp neigbors for a particular BGP instance
+    """
+    namespace = duthost.get_namespace_from_asic_id(asic_index)
+
+    # Convert the json formatted result of sonic-cfggen into bgp_neighbors dict
+    bgp_neighbors = json.loads(duthost.command("sudo sonic-cfggen {} -d --var-json {}".format('-n ' + namespace if namespace else '', "BGP_NEIGHBOR"))["stdout"])
+    cmds = []
+    for neighbor in bgp_neighbors.keys():
+        cmds.append('sudo sonic-db-cli {} CONFIG_DB del "BGP_NEIGHBOR|{}"'.format('-n ' + namespace if namespace else '', neighbor))
+    duthost.shell_cmds(cmds=cmds)
+
+    # Restart BGP instance on that asic
+    restart_bgp(duthost, asic_index)
+    
+    return bgp_neighbors
+
+def restore_bgp_neighbors(duthost, asic_index, bgp_neighbors):
+    """
+    Restore the bgp neigbors for a particular BGP instance
+    """
+    namespace = duthost.get_namespace_from_asic_id(asic_index)
+
+    # Convert the bgp_neighbors dict into json format after adding the table name.
+    bgp_neigh_dict = {"BGP_NEIGHBOR":bgp_neighbors}
+    bgp_neigh_json = json.dumps(bgp_neigh_dict)
+    duthost.shell("sudo sonic-cfggen {} -a '{}' --write-to-db".format('-n ' + namespace if namespace else '', bgp_neigh_json))
+
+    # Restart BGP instance on that asic
```

Comment:
can we do config_reload instead ?

Reply:
I did this because the current approach is quick and less time consuming than config reload, and in the test we can do it very specific to the BGP neighbors in that asic index.